make/env.cmake

```diff
@@ -156,7 +156,8 @@ function(ADD_CONDA_PACKAGE)
       DEPENDS conda ${CONDA_BIN}
       )
 
-    set(TARGET conda_${ADD_CONDA_PACKAGE_NAME})
+    # Allow specification of version and build, but strip it off for the package name
+    string(REGEX MATCH "^[^=]+" TARGET ${ADD_CONDA_PACKAGE_NAME})
```

Comment:
This change seems unrelated?

Reply:
It was an improvement during debugging. Do you want me to break out the one line change to a different PR?

Reply:
Ya